### PR TITLE
Assignment4 - 6ITA - Satyam Singh

### PR DESCRIPTION
**Pull request by mrSinghSatyam**
#231
Assignment4  Added

### CODE REVIEW

File: submissions/js/task_1_2.MD [hunk 1]
```diff
@@ -12,3 +12,4 @@
 |Vignesh Gaddam |2203031080131 |https://js-101-theta.vercel.app/ |https://github.com/mrvigneshgaddam/JS101 |
 |Dabhi Rajdeepsinh |2203031080069 |[Deployment URL](https://jsproject-nu.vercel.app/)| [Repository URL](https://github.com/Rajdeepsinh1410/JSPROJECT.git)|
 |Raj Patel |2203031080105 |[Deployment URL](https://js-five-beta.vercel.app/)| [Repository URL](https://github.com/RajPatel08/JS)|
+| Satyam Singh | 2203031080232 | https://vercel.com/satyam-singhs-projects-af8065ca/js-102 | https://vercel.com/satyam-singhs-projects-af8065ca/js-102 |
```
Comment (smaranjitghose) [inline submissions/js/task_1_2.MD:15]: This URL does not work

File: submissions/js/task_3_4.MD [hunk 2]
```diff
@@ -1,3 +1,3 @@
 | Student Name | Enrollment Number | Task 3 URL | Task 4 URL  | GitHub Repository URL |
 |---|---|---|---|---|
-|   |   |  |  |
+| Satyam Singh  | 2203031080232  | https://github.com/mrSinghSatyam/JS102/blob/master/count.html | https://github.com/mrSinghSatyam/JS102/blob/master/geometry.html |https://github.com/mrSinghSatyam/JS102 |
```
Comment (smaranjitghose) [inline submissions/js/task_3_4.MD:3]: This is part of Assignment 5

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: submissions/js/task_3_4.MD (hunk 2 above)
Comment (mrSinghSatyam) [reply submissions/js/task_3_4.MD]: #260 
Complete the changes

File: submissions/js/task_1_2.MD (hunk 1 above)
Comment (mrSinghSatyam) [reply submissions/js/task_1_2.MD]: URL is working

**Pull request merged**